tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -50,23 +47,31 @@ function HeaderComp() {
               <img src={closeIcon} alt="close" />
             </button>
             <ul className="usa-nav__primary usa-accordion">
-              <NavItem pathname={pathname} tabTitle="Welcome" href="/welcome" />
-              <NavItem
-                pathname={pathname}
-                tabTitle="Data Files"
-                href="/data-files"
-              />
-              <NavItem
-                pathname={pathname}
-                tabTitle="Profile"
-                href="/edit-profile"
-              />
-              {isSystemAdmin() && (
-                <NavItem
-                  pathname={pathname}
-                  tabTitle="Admin"
-                  href={`${process.env.REACT_APP_BACKEND_HOST}/admin/`}
-                />
+              {authenticated && (
+                <>
+                  <NavItem
+                    pathname={pathname}
+                    tabTitle="Welcome"
+                    href="/welcome"
+                  />
+                  <NavItem
+                    pathname={pathname}
+                    tabTitle="Data Files"
+                    href="/data-files"
+                  />
+                  <NavItem
+                    pathname={pathname}
+                    tabTitle="Profile"
+                    href="/edit-profile"
+                  />
+                  {isSystemAdmin() && (
+                    <NavItem
+                      pathname={pathname}
+                      tabTitle="Admin"
+                      href={`${process.env.REACT_APP_BACKEND_HOST}/admin/`}
+                    />
+                  )}
+                </>
```

Comment:
Prevent the NavItems from rendering based on the `authenticated` result from above